README.md

```diff
@@ -34,8 +45,24 @@ $ npm run dev -- --env
 
 To get an email from gov notify, you'll need to be added to the email list for testing on [gov notify](https://www.notifications.service.gov.uk/sign-in)
 
-## Skip email verify step
+## Skip email verify step  <a name="skip-email-verify-step"></a>
 
 On dev, preprod and locally you can skip the email verify step by going to the following url
 
 http://localhost:8081/nrm/start?token=skip
+
+
+## Acceptance Tests  <a name="acceptance-tests"></a>
+
+[Google Puppeteer](https://developers.google.com/web/tools/puppeteer/) is used to run our end-to end test
+scripts for the application. It is a Node library which provides a high-level API to control headless Chrome or Chromium over the DevTools Protocol. It can also be configured to use full (non-headless) Chrome or Chromium.
+
+### Running Acceptance Tests  <a name="running-acceptance-tests"></a>
+
+[Install & Run](#install-and-run)  the application then you can run the acceptance tests using the command below:
+
+```bash
+$ npm run start:local-acceptance
+```
+
+This will open up an instance of [Google Chrome](#https://www.google.com/chrome/) on your local machine for each test.
```

Comment:
is it worth also explaining how to run it with docker?

Reply:
No problem, I have a added a small overview of how to run with docker

Reply:
No problem I have added the comments in the new README.md commit